src/msa.js

```diff
@@ -145,7 +145,7 @@ const MSA = boneView.extend({
     this.removeViews();
 
     this.addView("stage",new Stage({model: this.seqs, g: this.g}));
-    this.el.setAttribute("class", "biojs_msa_div");
+    this.$el.addClass("class", "biojs_msa_div");
```

Comment:
ah damn I merged to quickly `addClass` doesn't need "class" anymore - will submit a fixup commit myself